### ISSUE TITLE
Add the Strebe 1995 projection

### ISSUE THREAD

**Issue by Veeno** [SciTools/cartopy#2094]
The title says it all for this feature request. [Strebe 1995](https://en.wikipedia.org/wiki/Strebe_1995_projection) is an equal-area projection with unique properties which I would really like to see added to cartopy.

**Comment by dopplershift** [comment]
Cartopy relies upon the [PROJ](https://proj.org/index.html) library (specifically the PyPROJ wrapper) for all of its projections. I wasn't able to find the projection in there with a quick search. If it actually is present in PROJ, then adding it to Cartopy should be a straightforward PR; if not, someone would first need to add it to PROJ.

**Comment by Veeno** [comment]
Indeed, it isn't included in [PROJ's available projections](https://proj.org/operations/projections/index.html). I will create an issue on the PROJ repository.
